### PR TITLE
CI with alpha channel and documentation

### PR DESCRIPTION
**Pull request by rafaelje**
### Changes description

Implement the new documentation path, remove the special jobs on the release branch now is the same process like a feature or fix, add Alpha channel to deploy, modify the master branch to understand if is a release or pre-release

Alpha
captura de pantalla 2018-09-11 a la s 10 35 51 a m

Master Pre-Release
captura de pantalla 2018-09-13 a la s 12 08 57 a m

Master Release
captura de pantalla 2018-09-13 a la s 10 15 02 a m

captura de pantalla 2018-09-13 a la s 10 15 37 a m

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@rafaelje |36|

### References

Closes #N/A
Related: https://github.com/flyve-mdm/android-mdm-agent/issues/562 
Depends on #N/A

### CODE REVIEW

**Comment by rafaelje** [comment]
Thank @Naylin15 I'll review the path to check your advise

**Comment by rafaelje** [comment]
@Naylin15 @ajsb85 @hectorerb 

This is a big PR with many changes about the documentation and the workflow to create a pre-release and release, what do you thing about merge this PR and create a new issue to solve the problem with the documentation?

**Comment by Naylin15** [comment]
Hi, @rafaelje 

I think it's a perfect idea

**Comment by rafaelje** [comment]
The documentation is an independent process and now is working but maybe can fail

**Comment by hectorerb** [comment]
Hi @rafaelje 
For me is fine!

File: ci/scripts/release.sh [hunk 1]
```diff
@@ -88,18 +91,9 @@ else
 fi
 
 # Update develop branch
+git add .
+git stash
```
[the remark this answers is not in the document]
Comment (rafaelje) [reply ci/scripts/release.sh]: I try with this command but not work

**Pull request merged**